jerry-core/ecma/operations/ecma-objects-general.c

```diff
@@ -313,7 +313,7 @@ ecma_op_general_object_define_own_property (ecma_object_t *object_p, /**< the ob
   JERRY_ASSERT (property_desc_p->is_writable_defined || !property_desc_p->is_writable);
 
   /* 1. */
-  ecma_extended_property_ref_t ext_property_ref;
+  ecma_extended_property_ref_t ext_property_ref = {{0},0};
```

Comment:
The Vera++ rule is not clever enough to recognize this. But the correct style would be `{ { 0 }, 0 }`
I suggest to use a define to workaround this problem.

```diff
diff --git a/jerry-core/ecma/base/ecma-globals.h b/jerry-core/ecma/base/ecma-globals.h
index 99b6da3..fa87e91 100644
--- a/jerry-core/ecma/base/ecma-globals.h
+++ b/jerry-core/ecma/base/ecma-globals.h
@@ -468,6 +468,11 @@ typedef struct
 #define ECMA_PROPERTY_SEARCH_DEPTH_LIMIT 128
 
 /**
+ * Empty extended property reference
+ */
+#define ECMA_EMPTY_EXTENDED_PROPERTY_REFERENCE { { 0 }, 0 }
+
+/**
  * Property reference. It contains the value pointer
  * for real, and the value itself for virtual properties.
  */
diff --git a/jerry-core/ecma/operations/ecma-objects-general.c b/jerry-core/ecma/operations/ecma-objects-general.c
index c24f356..5b479b2 100644
--- a/jerry-core/ecma/operations/ecma-objects-general.c
+++ b/jerry-core/ecma/operations/ecma-objects-general.c
@@ -313,7 +313,7 @@ ecma_op_general_object_define_own_property (ecma_object_t *object_p, /**< the ob
   JERRY_ASSERT (property_desc_p->is_writable_defined || !property_desc_p->is_writable);
 
   /* 1. */
-  ecma_extended_property_ref_t ext_property_ref;
+  ecma_extended_property_ref_t ext_property_ref = ECMA_EMPTY_EXTENDED_PROPERTY_REFERENCE;
   ecma_property_t current_prop;
 
   current_prop = ecma_op_object_get_own_property (object_p,
```